services/parser-database/parser.py

```diff
@@ -60,26 +61,12 @@ def identify_articles(pdf_text):
         list: article objects
     """
     articles = []
-    article_text = ""
-    article_count = 1
-    i = 0
-
-    while i < len(pdf_text):
-        if (pdf_text[i] == "artículo" or pdf_text[i] == "articulo") and (
-                pdf_text[i + 1] == str(article_count) + ".-"
-                or pdf_text[i + 1] == str(article_count) + "-"
-                or pdf_text[i + 1] == str(article_count) + "."):
-            logging.info("Article #" + str(article_count) + " recognized!")
-            articles.append(Article(article_count-1, article_text.strip()))
-            article_text = ""
-            article_count += 1
-            i += 1
-        else:
-            article_text += " " + pdf_text[i]
-            if i == len(pdf_text) - 1:
-                articles.append(Article(article_count-1, article_text.strip()))
+    i = 1
+    res = re.split(r'ART[ÍI]CULO *\d+ *\.?-?', pdf_text)
+    while i < len(res):
+        articles.append(Article(i, res[i].strip()))
```

Comment:
Cool, this is fine to merge as-is, but consider creating an issue to follow up in a later PR and capture the article number from the regex rather than using the value of `i`